sql_upgrade.php

```diff
@@ -69,81 +70,143 @@
 
 closedir($dh);
 ksort($versions);
+
+$res2 = sqlStatement("select * from lang_languages where lang_description = ?", array($GLOBALS['language_default']));
+for ($iter = 0; $row = sqlFetchArray($res2); $iter++) {
+    $result2[$iter] = $row;
+}
+
+if (count($result2) == 1) {
+    $defaultLangID = $result2[0]["lang_id"];
+    $defaultLangName = $result2[0]["lang_description"];
+    $direction = (int)$result2[0]["lang_is_rtl"] === 1 ? 'rtl' : 'ltr';
```

Comment:
just note this lang_is_rtl field is rather new, so just make sure logic is ok if it doesn't exist